src/qt/sendcoinsdialog.cpp

```diff
@@ -262,15 +262,14 @@ void SendCoinsDialog::on_sendButton_clicked()
     if (CoinControlDialog::coinControl->fSplitBlock)
         CoinControlDialog::coinControl->nSplitBlock = int(ui->splitBlockLineEdit->text().toInt());
 
-    QString strFunds = tr("using") + " <b>" + tr("anonymous funds") + "</b>";
+    QString strFunds = "";
     QString strFee = "";
     recipients[0].inputType = ALL_COINS;
-    strFunds = tr("using") + " <b>" + tr("any available funds (not recommended)") + "</b>";
 
     if (ui->checkSwiftTX->isChecked()) {
         recipients[0].useSwiftTX = true;
         strFunds += " ";
-        strFunds += tr("and SwiftX");
+        strFunds += tr("using SwiftX");
```

Comment:
would splitting the string make more sense? "SwiftX" is a proper name so I don't think we want it translated, and the string "using" is translated on it's own already, so...something like:
```c++
strFunds += tr("using") + "SwiftX";
```

or if we did happen to want "SwiftX" passed to transifex:
```c++
strFunds += tr("using") + " " + tr("SwiftX");
```